drivers/base/bootvid/common.c

```diff
@@ -272,6 +327,22 @@ RleBitBlt(
 
 /* PUBLIC FUNCTIONS ***********************************************************/
 
+/*
+ * @implemented
+ */
```

Comment:
Is this necessary?

Reply:
Yep, these comments are not terrible. It would be better to have an actual doxygen documentation instead (but this is off-topic for this PR).